editor/src/components/inspector/sections/component-section/variables-in-scope-utils.ts

```diff
@@ -395,14 +395,23 @@ export function useVariablesInScopeForSelectedElement(
     'useVariablesInScopeForSelectedElement selectedViewPath',
   )
 
+  return useVariablesInScopeForElementPath(elementPath, selectedViewPath, propertyPath, mode)
+}
+
+export function useVariablesInScopeForElementPath(
+  elementPath: ElementPath | null,
+  selectedViewPath: ElementPath | null,
```

Comment:
does anything break if you unify elementPath and selectedViewPath into a single `elementPath`? I think in the original code it was an accident that the hook took a selectedView, but also called `useEditorState` for a selectedView

Reply:
yeah I also thought about it. I fixed it in https://github.com/concrete-utopia/utopia/pull/5789/commits/e022627adbcdcb07cf19678ba7fe94d65bfb0d5a, it seems like it's still fine?

Reply:
awesome!